main/mio.c

```diff
@@ -43,6 +43,13 @@
 #include <stdlib.h>
 #include <limits.h>
 
+#ifndef WIN32
+#ifndef _MSC_VER
```

Comment:
Isn't this line redundant?
If `WIN32` isn't defined, `_MSC_VER` won't be defined.

Reply:
Oh, I see. Thank you!